src/content/docs/dev/send-tokens/sui-its.mdx

```diff
@@ -0,0 +1,502 @@
+    } else {
+        coin::take(self.balance.borrow_mut(), amount, ctx)
+    }
+}
+```
+
+
+### Take Balance
+When the coin is being out of the Sui ecosystem, the Management module will trigger the `take_balance()` function. This function updates the [flow limit](#flow-limit) for an outgoing transfer and then either burns the coins (if minting capability is available) or merges the deducted amount into the stored balance. It returns the numeric amount (as a `u64`) that was taken.
+
+
+```move
+public(package) fun take_balance<T>(self: &mut CoinManagement<T>, to_take: Balance<T>, clock: &Clock): u64 {
+    self.flow_limit.add_flow_out(to_take.value(), clock);
+    let amount = to_take.value();
+    if (has_capability(self)) {
+        self.burn(to_take);
+    } else {
+        self.balance.borrow_mut().join(to_take);
+    };
+    amount
+}
+```
+
+### Set Flow Limit
+To set a specific [flow limit](#flow-limit) amount you can trigger the `set_flow_limit()` function.
+
+The function takes three parameters:
+
+1. `coin_management` (`self`): The module that will be updated once the `flow_limit` is set
+1. `channel`: A reference to the `Channel` object to derive the caller’s address. The address is checked to ensure that only the authorized operator can change the `flow_limit`.
+1. `flow_limit`: An optional unsigned 64-bit integer representing the new flow limit
+
+```move
+public(package) fun set_flow_limit<T>(self: &mut CoinManagement<T>, channel: &Channel, flow_limit: Option<u64>) {
+    assert!(self.operator.contains(&channel.to_address()), ENotOperator);
+    self.set_flow_limit_internal(flow_limit);
+}
+```
+
+### Roles
+
+The Coin Management module has set roles that can handle specific functionality
+
+#### Treasury Capability
+A special permission (represented as a `TreasuryCap`) that, when present, enables minting and burning of tokens. A CoinManagement instance created with this capability can adjust the token supply.
+
+#### Distributor
+An address set within the CoinManagement instance that is authorized to perform minting (and sometimes burning) operations. Only CoinManagement instances with a treasury capability can add a distributor.
+
+#### Operator
+An address authorized to update [flow limits](#flow-limit) for token transfers. When setting a new flow limit, the module verifies that the caller’s channel address matches the stored operator address to ensure only the operator can make that change.
+
+
+## Flow Limit
+Flow Limits represent the volume of a coin that can be transferred in/out of Sui via ITS. This limit plays a critical role in maintaining network integrity and security. When coins are sent out of Sui. The flow limit logic can be found in its own [Flow Limit Module](https://github.com/axelarnetwork/axelar-cgp-sui/blob/main/move/interchain_token_service/sources/types/flow_limit.move)
+
+The module has several pieces of functionality that are triggered by the [Coin Management](#coin-management) module
+
+### Add Flow In
```

Comment:
no need to document this. Also no need to document any public(package) functions as they cannot be used by users.